esptfa_arima/Authentication/views.py

```
@@ -20,6 +21,11 @@
 from .serializers import UserSerializer, TeacherSerializer, StudentSerializer
 from .services import register_user, login_user, process_manual_import, process_csv_import
 import pandas as pd
+from rest_framework import status, viewsets
```

Ruff (0.14.10)

Redefinition of unused `status` from line 15

Remove definition: `status`

(F811)